dev-util/github-cli/github-cli-0.9.0-r1.ebuild

```diff
@@ -294,4 +294,10 @@ src_compile() {
 src_install() {
 	dobin bin/gh
 	dodoc README.md
+
+	make manpages
```

Comment:
the `make manpages` triggers further Go compilation here; the makefile needs changes to do that during compile instead.